src/main.cpp

```diff
@@ -209,8 +209,10 @@ int main(int argc, char *argv[])
     std::shared_ptr<sf::RenderWindow> win;
     if (settings->value("video.fullscreen", true))
     {
+        logger->debug("Using desktop window.");
+
         win = std::make_shared<sf::RenderWindow>(
-            sf::VideoMode(static_cast<unsigned int>(width), static_cast<unsigned int>(height)),
+            sf::VideoMode::getDesktopMode(),
```

Comment:
Does this change the resolution of the game? I wanted (more or less) the same things to be visible when you start the game in fullscreen or in windowed mode.

Reply:
Well the previous change completely breaks on ubuntu at 1024x768 to where I have to run xrandr from outside of X Windows to reset screen resolution, otherwise my whole desktop is black.